File: Week_0.1/q19.c
#include <stdio.h>

int main()
{
    printf("Enter the annual income ");
    float income;
    scanf("%f", &income);
    float tax;

    if (income <= 250000)
    {
        printf("Not tax \n");
    }
    else if (income > 250000 && income <= 500000)
    {
        tax = (income * 5) / 100;
        printf("Income tax amount = %0.2f\n", tax);
    }
    else if (income > 500000 && income <= 1000000)
    {
        tax = (income * 20) / 100;
        printf("Income tax amount = %0.2f\n", tax);
    }
    else if (income > 1000000 && income <= 5000000)
    {
        tax = (income * 30) / 100;
        printf("Income tax amount = %0.2f\n", tax);
    }
}

File: Week_3/q19.js
var readlineSync = require('readline-sync')

const income = readlineSync.question("Enter the annual income ");
let tax;

if (income <= 250000) {
    console.log("No tax");
}
else if (income > 250000 && income <= 500000) {
    tax = (income * 5) / 100;
    console.log("Income tax amount = " + tax);
}
else if (income > 500000 && income <= 1000000) {
    tax = (income * 20) / 100;
    console.log("Income tax amount = " + tax);
}
else if (income > 1000000 && income <= 5000000) {
    tax = (income * 30) / 100;
    console.log("Income tax amount = " + tax);
}
